python/mlcroissant/pyproject.toml

```diff
@@ -50,6 +50,7 @@ dev = [
   "mlcroissant[git]",
   "mlcroissant[image]",
   "mlcroissant[parquet]",
+  "mlcroissant[video]",
```

Comment:
Remove this line

Reply:
Sorry, I somehow didn't include this in the commit